pages/books/listItems.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import ListSubheader from '@mui/material/ListSubheader';
+import DashboardIcon from '@mui/icons-material/Dashboard';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import PeopleIcon from '@mui/icons-material/People';
+import BarChartIcon from '@mui/icons-material/BarChart';
+import LayersIcon from '@mui/icons-material/Layers';
+import AssignmentIcon from '@mui/icons-material/Assignment';
+
+export const mainListItems = (
```

Comment:
this does not seem to be a page, this could be better off in the `components` directory. the next.js `pages` directory is tied to the routing so we have to be careful with the components that we add, see the docs for more details - https://nextjs.org/docs/api-routes/introduction

Reply:
okay will do it

Reply:
done